### PR TITLE
Remove legacy style, it should be more modern now.

### PR DESCRIPTION
**Pull request by yanzhiwei147**
1. More modern style appearance.
2. It can fix the #63 issue.

### CODE REVIEW

File: Pod/Classes/JSBadgeView.m [hunk 1]
```diff
@@ -46,27 +46,6 @@ of this software and associated documentation files (the "Software"), to deal
...
-            isUIKitFlatMode = (NSVersionOfLinkTimeLibrary("UIKit") >> 16) >= UIKitVersionNumber_iOS_7_0;
-        }
-    });
-
-    return isUIKitFlatMode;
-}
-
 @implementation JSBadgeView
 
 + (void)applyCommonStyle
```
Comment (JaviSoto) [inline Pod/Classes/JSBadgeView.m]: We can now just name this `applyStyle`

**Review by JaviSoto — APPROVED** [review]
👍  Yeah this is totally reasonable!

**Comment by JaviSoto** [comment]
I should add however, that with the new style, this class can be massively simplified. There's no reason to render the view programmatically in the CPU via `-drawRect:`. It could be much more simply implemented with a UILabel subview. And with auto-layout, the sizing and positioning code would be a lot simpler as well.

File: Pod/Classes/JSBadgeView.m (hunk 1 above)
Comment (yanzhiwei147) [reply Pod/Classes/JSBadgeView.m]: Yeah i can't agree more!

**Comment by yanzhiwei147** [comment]
Yes, it should be refactor, so we can implement with AutoLaout, but i think it's the other question.

**Comment by JaviSoto** [comment]
Thank you!

**Comment by yanzhiwei147** [comment]
This PR merged now, so we trunk version 2.0.0 to cocoapods specs now?

**Pull request merged**